homeassistant/components/zodiac/translations/sensor.en.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
these translations should be done via lokalise rather than directly (will be overrriden nightly)